### PR TITLE
Add docs

### PR DESCRIPTION
**Pull request by bobbysn0w**
As requested in #120 

Contributors can edit the docs by adding a PR.

Access the docs at yalu102.rtfd.io

Will update parent repo to this one if approved (so PRs submitted here for the docs actually update the doc website)

### CODE REVIEW

**Comment by nullpixel1** [comment]
I don't like commit https://github.com/kpwn/yalu102/pull/131/commits/36ce14edd91b73106d462979c57a45aff90cb411, I see that discord link.

File: docs/index.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ Welcome to yalu102's documentation. Visit the GitHub Project [here](https://gith
 
 This is a "work in progress" Jailbreak designed for pre-i7 devices running iOS 10.2.
 
-It was created by [qwertyoruiopz](https://twitter.com/qwertyoruiopz) and [marcograssi](https://github.com/kpwn/yalu102/blob/master/marcograss)
+It was created by [qwertyoruiopz](https://twitter.com/qwertyoruiopz) and [marcograssi](https://github.com/kpwn/yalu102/blob/master/marcograss). This documentation was created, and is mantained, by [JailbreakHub](https://jailbreakhub.org). It is not affilated with Qwerty or Marcograss.
```
Comment (nullpixel1) [inline docs/index.md]: This is a community maintIned wiki

**Comment by AppleBetas** [comment]
@nullpixel1 Yep. This is a community project, if you feel the need to credit yourself, just don't contribute.

File: docs/index.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+yalu103 - Alpha | [Link](https://yalu.qwertyoruiop.com/yalu102_alpha.ipa) | 2fe14f1c1e1a0d26203bbb123f6747a978dd2b4f
+YaluB3 | [Link](https://yalu.qwertyoruiop.com/mach_portal+yalu-b3.ipa) | f34ba1bebf02b265a871158e778596ee2ff1547a69fdb687d911897b7b7b22c5
+
+## Guides
+
+* [Checking your SHA1](sha1/)
+
+## Public Wiki
+
+These docs were brought to you by [JailbreakHub](redirect/)
```
Comment (nullpixel1) [inline docs/index.md]: The docs are a community project, done publicly

File: docs/index.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+## Have a problem or suggestion?
+
+* Make a [Pull Request](https://github.com/iOptimistic/yalu102/pulls)
+
+* Contact the [WebMaster](mailto:bobby@jailbreakhub.org?Subject=Yalu102%20Suggestions)
```
Comment (nullpixel1) [inline docs/index.md]: Please remove thus email, creating an issue on GitHub would be better

File: docs/redirect.html [hunk 4]
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE HTML>
```
Comment (nullpixel1) [inline docs/redirect.html]: Why is any of this page needed?

**Comment by ninjaprawn** [comment]
Is any of this really needed? GitHub repos have an inbuilt wiki feature...

File: docs/index.md (hunk 3 above)
Comment (Acelogic) [inline docs/index.md]: Agreed

**Comment by nullpixel1** [comment]
True, if we're making a wiki, that is the way to go. @ninjaprawn 

This PR seems like a whole self advertisement thing to me.

File: docs/sha1.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
+# How To Verify Your Install
+
+The SHA1 checksums are hashes developed by a Secure Hash Algorithm. ELI5: Every file has a specific SHA1 checksum that is changed every time the file is changed. This is to make sure your download isn't corrupt or contains harmful code.
```
Comment (stal888) [inline docs/sha1.md]: > SHA1 checksums are hashes developed by a Secure Hash Algorithm

SHA-1 was designed by the NSA. It didn't invent itself.

File: docs/index.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+
+* [Checking your SHA1](sha1/)
+
+## Public Wiki
+
+These docs were brought to you by [JailbreakHub](redirect/)
+
+## Have a problem or suggestion?
+
+* Make a [Pull Request](https://github.com/iOptimistic/yalu102/pulls)
```
Comment (stal888) [inline docs/index.md]: This link does not go to a public repo.

File: docs/index.md (hunk 3 above)
Comment (Legitgamer264) [inline docs/index.md]: There is not a need for this at all

Comment (Legitgamer264) [inline docs/index.md]: There is no point in mentioning the site. as Null said, just create a Issue report

**Comment by meyer9** [comment]
:-1: This seems unneeded
